GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -274,6 +274,7 @@ private ContextMenuSelectionInfo GetSelectionInfo()
             bool isAnyWorkTree = DiffFiles.SelectedItems.Any(item => item.Staged == StagedStatus.WorkTree);
             bool isAnySubmodule = DiffFiles.SelectedItems.Any(item => item.IsSubmodule);
             bool singleFileExists = isExactlyOneItemSelected && File.Exists(_fullPathResolver.Resolve(DiffFiles.SelectedItem.Name));
+            bool isAnyFileExist = DiffFiles.SelectedItems.Any(item => File.Exists(_fullPathResolver.Resolve(item.Name)));
```

Comment:
isAnyExistingFile
Even my English have a problem with that
This adds I/O when opening menus, is this OK?